pf/tfgen/main.go

```diff
@@ -79,50 +79,62 @@ func Main(provider string, info tfbridge.ProviderInfo) {
 func MainWithMuxer(provider string, infos ...tfbridge.Muxed) {
 	ctx := context.Background()
 
-	// It is safe to pass infos[0].GetInfo() to MainWithCusomGenerate because it is
-	// only used when passed to GenerateOptions.
+	contract.Assertf(len(infos) == 2,
```

Comment:
Oof a bit brutal, so the idea is to change the signature later? note the code expect the right order also not just len=2.

Reply:
Fixed by merging in #997.